config/getting-started/Application Developer/Getting Started.adoc

```diff
@@ -0,0 +1,14 @@
+== Getting Started for Application Developers
+
+'''''
+
+*https://docs.devfile.io/devfile/2.1.0/user-guide/api-reference.html[Devfile
+specification]*
+
+*Tools that provides devfile support:*
+
+* https://odo.dev/[odo]
+* https://medium.com/eclipse-che-blog/devfile-v2-and-ide-plug-ins-in-eclipse-che-7a560ae724b1[Eclipse
+Che]
```

Comment:
Links seem to be working, but can we have the Link description be in the same line rather than being on new line on this doc?

same for all the newly created adoc files

Reply:
fixed